src/molecules/footer-menu/FooterMenu.html

```diff
@@ -0,0 +1,44 @@
+<div :class="['a-footer-menu', isLargeScreen && 'a-footer-menu--screen-m']">
+  <a-collapse
+    class="a-footer-menu__list"
+    button-class="a-footer-menu__button"
+    content-class="a-footer-menu__content"
+    v-for="item in menu"
+    :key="item.id"
+    :visible="isLargeScreen"
+    :title="item.title"
+    :id="item.id"
+  >
+    <template #button="{ visibility, ariaControls, toggle }">
```

Comment:
IE alert 😃 

For now we need to avoid this approach :)

Reply:
Example

https://github.com/SnowdogApps/alpaca-ui/blob/f6b8e7a987fe91fe261ac3069201a4fc112ccc32/src/molecules/quantity-update/QuantityUpdate.stories.js#L100

Reply:
good point ;)